src/main/java/wedlog/address/ui/MainWindow.java

```diff
@@ -223,7 +218,6 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
-            setStatisticsPanel();
```

Comment:
the current guest and vendor counts displayed at the top of each panel don't get updated when guests/vendors are added and deleted. think we may need to have a setGuestListPanel and a setVendorListPanel method that get called  in executeCommand to update the numbers each time